### PR TITLE
George export disable

### PR DESCRIPTION
**Pull request by Georgehe4**
Fixed splitter size issue post-vulcanization.
@devoncarew 
Had to set the height of the outside divs to 100%

### CODE REVIEW

File: web/embed-dart.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
   </head>
 
   <body class="fullbleed vertical layout" unresolved>
-    <div class="flex layout horizontal">
+    <div id = "main" class="flex layout horizontal">
```
Comment (devoncarew) [inline web/embed-dart.html]: nit: `id="main"` (no spaces)

File: web/embed-html.html [hunk 2]
```diff
@@ -41,7 +41,7 @@
 
   <body class="fullbleed vertical layout" unresolved>
     <menu-bar-icons></menu-bar-icons>
-    <div class="flex layout horizontal">
+    <div id = "main" class="flex layout horizontal">
```
Comment (devoncarew) [inline web/embed-html.html]: same

File: web/embed_style.html [hunk 3]
```diff
@@ -46,11 +50,16 @@
...
+    height: 70%;
   }
   
   #leftPanel {
     width: 60%;
+    height: 100%;
+  }
+  
+  #rightPanel {
+    height:100%;
```
Comment (devoncarew) [inline web/embed_style.html]: nit: `: 100%`

**Comment by devoncarew** [comment]
lgtm!

@Georgehe4, feel free the TBR work while I'm out. I'll review after the commit, and you won't be blocked while I'm on vacation.

**Comment by Georgehe4** [comment]
Merging.

**Pull request merged**